src/components/modalBreadcrumb/modalBreadcrumb.vue

```diff
@@ -1,9 +1,19 @@
 <template>
   <nav :class="['apos-breadcrumb', classObj]" aria-label="breadcrumb">
     <ol class="apos-breadcrumb__items">
-      <li v-for="(item, index) in items" :key="index" :class="`apos-breadcrumb__item ${modifier}`">
-        <component :is="item.href ? 'a' : 'span'" :href="item.href">{{ item.label }}</component>
-        <ChevronRightIcon class="apos-breadcrumb__chevron" :size="13" v-if="index !== last" />
+      <li
+        v-for="(item, index) in items" :key="index"
+        :class="`apos-breadcrumb__item ${modifier}`"
+      >
+        <component
+          :is="item.href ? 'a' : 'span'" :href="item.href"
```

Comment:
these should probably be buttons and not anchors

Reply:
Good point.